man/mono.1

```diff
@@ -860,9 +860,11 @@ Instruct the runtime on the number of times that the method specified
 by --compile (or all the methods if --compile-all is used) to be
 compiled.  This is used for testing the code generator performance. 
 .TP 
-\fB--stats\fR
+\fB--stats=[method]\fR
 Displays information about the work done by the runtime during the
-execution of an application. 
+execution of an application. If a method (namespace.name:methodname)
+is specified, it will display that information when the method is run
```

Comment:
```suggestion
is specified, it will display that information when the method is jitted
```
```suggestion
is specified, it will display that information when the method is run for the first time
```

Not sure on the exact wording, but from the looks of it, it's not run every time the method is run.

Reply:
It's the interpreter as well as the JIT, but yeah I should probably tweak the wording. I'll do that when I fix the silly build error later today.

Reply:
I just put "when the method is first run", which is hopefully clear enough while staying concise.